sigma-ser/src/vlq_encode.rs

```diff
@@ -232,7 +232,7 @@ pub trait ReadSigmaVlqExt: io::Read + io::Seek {
 
     /// Read a vector of bits with the given size
     fn get_bits(&mut self, size: usize) -> Result<Vec<bool>, VlqEncodingError> {
-        let byte_num = (size + 7) / 8;
+        let byte_num = size.div_ceil(8);
```

Comment:
is it the same?

Reply:
Yes, all the code changes are to address lints added in newer compiler versions. https://rust-lang.github.io/rust-clippy/master/index.html#manual_div_ceil